tests/console_test.go

```diff
@@ -56,8 +56,9 @@ var _ = Describe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@redha
 
 	RunVMIAndWaitForStart := func(vmi *v1.VirtualMachineInstance) {
 		By("Creating a new VirtualMachineInstance")
-		vmi.ObjectMeta.Namespace = util.NamespaceTestDefault
-		Expect(virtClient.RestClient().Post().Resource("virtualmachineinstances").Namespace(util.NamespaceTestDefault).Body(vmi).Do(context.Background()).Error()).To(Succeed())
+		createdVmi, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
```

Comment:
I guess [this](https://github.com/kubevirt/kubevirt/pull/7114/commits/3916774cd0c1b7498a02393060e88613a38a43f9) commit can be taken out as you are removing this function in follow up.

Reply:
You are right; I kept my own narrative in the commits, instead of adapting it to the reader.
I don't know how strongly you feel about this, but next time please use /hold to avoid automatic merge if you believe it is important enough.

Reply:
One additional commit is not as bad.